docker/README.md

```diff
@@ -0,0 +1,12 @@
+# Docker environment
```

Comment:
Add a short sentence that says this docker environment is only for developing and testing Lego.
Not everyone should setup this environment, and this isn't how LegoManager should be deployed.

Reply:
Fixed